docs/widgets/video-widget.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 📹 Video
+tags:
+  - Widgets
+  - Integration
+  - Configuration
+---
+
+
+The video stream Widget is a web video player for an HTML5. It supports HTML5 video and Media Source Extensions.
+It can display content from web camera, or just watching your favorite stream.
+
+---
+
+## Adding the widget
+Please check out our documentation on [how to add a widget](/docs/introduction/after-the-installation#adding-widgets).
+
+---
+
+## Configuration
+
+| Configuration         | Description | Values | Default Value |
+| --------------------- | ----------- | ------ | ------------- |
+| Feed URL | Video Url. | - | - |
+| Auto play | Auto play when Homarr load. | yes / no | yes |
+| Muted audio | Mute audio. | yes / no | yes |
+| Video player controls | Add player controls. | yes / no | no |
+
+## Using the widget
+
+You can integrate any stream or video, that is being supported by [video js](https://videojs.com/).
+
+:::tip
+You can use RTSPtoWeb to streem security camera.
+
+:::
+
+### [RTSPtoWeb](https://github.com/deepch/RTSPtoWeb)
+converts your RTSP streams to formats consumable in a web browser.
+
+#### Installation
+
+* To install using Portainer Stacks, simply create a new stack and paste the following code into it.
+
+* To install using Docker Compose, simply create a file called ``docker-compose.yml`` and paste the following code into it.
+
+```yml title="docker-compose.yml"
+version: '3.3'
+services:
+  deepch:
+    container_name: rtsp-to-web
+    image: ghcr.io/deepch/rtsptoweb:latest
+    network_mode: host
+    ports:
+      - 8083:8083
+    restart: unless-stopped
+```
+
+Then, run ``docker-compose up -d`` in the same directory. This will start the container in the background.
+
+#### Using the RTSP streem
+* Open the url http://localhost:8083.
+* Configure your cameras.
+* Run this command on cmd / terminal - curl http://demo:demo@localhost:8083/streams (you need to change localhost to your server url).
+* Copy camera {stream_name}
+* Paste this url in Widget feed url - http://localhost:8083/stream/{stream_name}/channel/{channel_name}/hls/live/index.m3u8
```

Comment:
```suggestion
```

Remove this because it is not a product supported natively by Homarr. This does not belong in Homarr docs.